keps/sig-network/1669-graceful-termination-local-external-traffic-policy/README.md

```diff
@@ -189,34 +194,25 @@ Try to be as paranoid as possible - e.g., what if some components will restart
 mid-rollout?
 -->
 
-TBD for beta.
+Roll out can fail if there are pods receiving traffic during termination but are unable to handle it.
```

Comment:
Sorry - can't comment on unchanged lines:

L182 - have those been added? If so, can you please link them?